src/rgw/rgw_common.h

```diff
@@ -137,6 +137,10 @@ using ceph::crypto::MD5;
 
 #define RGW_ATTR_APPEND_PART_NUM    RGW_ATTR_PREFIX "append_part_num"
 
+/* to be used for sync of cloudtiered objects */
+#define RGW_ATTR_CLOUD_TIER_TYPE    RGW_ATTR_PREFIX "cloud_tier_type"
+#define RGW_ATTR_CLOUD_TIER_CONFIG    RGW_ATTR_PREFIX "cloud_tier_config"
```

Comment:
these attrs are 'special' in that they aren't stored as xattrs in the head object. that's probably worth mentioning in this comment